geolocation.py

```diff
@@ -106,6 +109,51 @@ def get_country_from_location(location_string):
     if location_string is None:
         return "None"
 
+    location_string = location_string[0].upper() + location_string[1:]
```

Comment:
@kdobolyi, does capitalize() fit your use case here? See: https://docs.python.org/3/library/stdtypes.html#str.capitalize - Feel free to ignore if it does not.

Reply:
almost -- that one will force the rest to be lowercase, which in some cases may not be what we want :-) (I could probably write cleaner code that just lowercases everything, but that's a rewrite for another day :-))